package.json

```diff
@@ -56,14 +67,19 @@
     "eslint-plugin-react": "^7.7.0",
     "eslint-plugin-standard": "^3.0.1",
     "exports-loader": "^0.7.0",
+    "express": "^4.16.3",
     "file-loader": "^1.1.11",
     "html-webpack-plugin": "^3.0.6",
     "if-env": "^1.0.4",
+    "karmatic": "^1.1.7",
     "loader-utils": "^1.1.0",
     "mini-css-extract-plugin": "^0.3.0",
+    "mocha": "^5.2.0",
     "node-sass": "^4.7.2",
     "optimize-css-assets-webpack-plugin": "^4.0.0",
+    "prettier": "^1.12.1",
```

Comment:
Is this one from me? I think this can go 😅

Reply:
must be, haha